src/HelperPluginManager.php

```diff
@@ -174,4 +183,29 @@ public function validate(mixed $instance): void
             );
         }
     }
+
+    /**
+     * @template InstanceParam of HelperInterface
+     * @param class-string<InstanceParam>|string $id Service name of plugin to retrieve.
+     * @return ($id is class-string<InstanceParam> ? InstanceParam : InstanceType)
+     */
+    public function get(string $id): mixed
+    {
+        /** @psalm-var InstanceType $plugin Unfortunately this type needs forcing */
+        $plugin = parent::get($id);
+        if ($plugin instanceof StatefulHelperInterface) {
+            $this->statefulHelpers[spl_object_hash($plugin)] = $plugin;
```

Comment:
`spl_object_id()` is exists since php 7.2, I think we can use `spl_object_id()`.

Refs:

- https://github.com/php/php-src/pull/2611
- https://developer.wordpress.com/2023/08/24/speedier-php-execution-in-wordpress-6-3/

**Benchmark**

**Before** **_call in loop took: 13.639ms_**

https://3v4l.org/4rUL4

**After**  **_call in loop took: 4.835ms_**

https://3v4l.org/jKtO6

Reply:
That's a pretty big difference 😃